src/index.html

```diff
@@ -11,7 +11,121 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="#" method="post">
+      <fieldset class="form-field">
+        <legend>Personal information</legend>
+        <label class="field">
+            Surname:
+          <input type="text" autocomplete="off">
+        </label>
+        <label class="field">
+            Name:
+          <input type="text" autocomplete="off">
```

Comment:
```suggestion
          Name:
          <input type="text" autocomplete="off">
```
no need to add these indents, fix all cases